assets/js/googleSearch.js

```diff
@@ -1,15 +0,0 @@
-var autocomplete;
-
-function initAutocomplete() {
-  autocomplete = new google.maps.places.Autocomplete(
-  (document.getElementById('autocomplete')),
-  {types: ['geocode']});
-  };
```

Comment:
Semicolons after function declarations are unnecessary. An assignment of a function expression to a variable does require semicolons. 

```
function initMap() {
} // No semicolon here

var initMap = function () {
}; // Semicolon belongs here
```

Keep in mind that statements require semicolons and that an assignment is a statement.